Backport of https://gitlab.gnome.org/GNOME/pango/-/commit/89ca7a4299b873746e3b6c1e1260ee8430b63fd4
From 89ca7a4299b873746e3b6c1e1260ee8430b63fd4 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Wed, 10 Jun 2020 09:41:03 -0400
Subject: [PATCH] Fix a crash in gtk2

Apparently, gtk2 assumes that calling pango_layout_set_attributes
guarantees that the attribute list gains a ref.

Fixes: #479
---
 pango/pango-layout.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/pango/pango-layout.c b/pango/pango-layout.c
--- a/pango/pango-layout.c
+++ b/pango/pango-layout.c
@@ -689,11 +689,6 @@
   if (!attrs && !layout->attrs)
     return;
 
-  /* Also both empty */
-  if (!_pango_attr_list_has_attributes (layout->attrs) &&
-      !_pango_attr_list_has_attributes (attrs))
-    return;
-
   old_attrs = layout->attrs;
 
   /* We always clear lines such that this function can be called
